src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,102 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  nextPage: () => void;
+  prevPage: () => void;
+  visibleItems:string[];
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  nextPage,
+  prevPage,
+  visibleItems,
+  onPageChange,
+}) => {
+  const pageNumbers = [];
+
+  // eslint-disable-next-line no-plusplus
+  for (let i = 1; i < Math.ceil(total / perPage) + 1; i++) {
+    pageNumbers.push(i);
+  }
```

Comment:
You can create pages with helper `getNumbers`

```suggestion
  const pages = getNumbers(1, total / perPage);
```